app_dev.yaml

```diff
@@ -113,6 +113,18 @@ handlers:
   secure: always
   expiration: "0"
 
+# PRODUCTION/DEVELOPMENT DYNAMIC
+# Note: Redirection is disabled (using 'secure: never') for requests that start
+#   with '/task/.*', mainly because Taskqueue API sends requests with HTTP and
+#   rejects them when we try to redirect them to HTTPS.
+- url: /task/.*
+  script: auto
+  secure: never
+- url: /.*
```

Comment:
Why did this not need to be added before?

Reply:
By default all stuff that is not handled in previous options goes to `/.*`, just I wanted to mention it explicitly.